daisy/step_resize_disks.go

```diff
@@ -0,0 +1,100 @@
+//  Copyright 2017 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package daisy
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"google.golang.org/api/compute/v1"
+)
+
+// ResizeDisks is a Daisy ResizeDisks workflow step.
+type ResizeDisks []*ResizeDisk
+
+// ResizeDisk is used to resize a GCE disk.
+type ResizeDisk struct {
+	compute.DisksResizeRequest
+	// Name of the disk to be resized
+	Name          string
+	Project, Zone string
```

Comment:
I think all you should have here is Name, or Disk (see start/stop instance step and deleteresources)
I think what we've settled on is to allow either the daisy name or the uri of the resource.